src/main/java/teammates/logic/api/Logic.java

```diff
@@ -1510,13 +1510,17 @@ public FeedbackQuestionAttributes getFeedbackQuestion(String feedbackQuestionId)
      */
     public String getFeedbackSessionResultSummaryAsCsv(String courseId,
                                                        String feedbackSessionName,
-                                                       String instructorEmail)
+                                                       String instructorEmail,
+                                                       String filterText)
             throws EntityDoesNotExistException, ExceedingRangeException {
         
         Assumption.assertNotNull(ERROR_NULL_PARAMETER, courseId);
         Assumption.assertNotNull(ERROR_NULL_PARAMETER, feedbackSessionName);
         
-        return feedbackSessionsLogic.getFeedbackSessionResultsSummaryAsCsv(feedbackSessionName, courseId, instructorEmail);
+        return feedbackSessionsLogic.getFeedbackSessionResultsSummaryAsCsv(feedbackSessionName,
+                                                                           courseId,
+                                                                           instructorEmail,
+                                                                           filterText);
```

Comment:
Where possible, try to avoid stacking of parameters eg try:

```
return feedbackSessionsLogic.getFeedbackSessionResultsSummaryAsCsv(
        feedbackSessionName, courseId, instructorEmail, filterText);
```

if the method name is too long. This applies to all methods which you have changed here; methods which you didn't change can remain as is.